src/utils/index.ts

```diff
@@ -407,6 +403,19 @@ export async function getMulticallProvider(
       address: "0x149bBb210051851016F57a2824C0444f642833a6",
       block: 0,
     }
+  } else if (chainId === ChainId.AURORA) {
+    ethcallProvider.multicall3 = {
```

Comment:
Lets use the official multicall3 deployment if possible.
https://github.com/mds1/multicall